docs/conf.py

```diff
@@ -58,7 +58,7 @@
 master_doc = "index"
 
 # General information about the project.
-project = "Nox"
+project = "nox"
```

Comment:
I consider this should keep as `Nox`, because it is a project?

Reply:
If no one disagrees, this is my suggestion:

```suggestion
project = "Nox"
```

Reply:
@DiddiLeija, I applied your suggestion to the code. And I'm open to any other suggestions.